synfig-core/src/synfig/time.cpp

```diff
@@ -326,7 +326,7 @@ Time::get_string(float fps, Time::Format format)const
 Time
 Time::round(float fps)const
 {
-	// the aim is to make results for the same frame are absolutelly idential
+	// the aim is to make results for the same frame absolutely identical
```

Comment:
Should this be `the aim is to make sure the results for the same frame are absolutely identical` ?

Reply:
It's ok to leave as "the aim is to make results for the same frame absolutely identical" ^__^